src/shogun/lib/DynamicArray.h

```diff
@@ -621,21 +774,13 @@ template <class T> class CDynamicArray :public CSGObject
 		virtual void init()
 		{
 			set_generic<T>();
+			T* head = m_array.data();
+			m_parameters->add_vector(
+			    &head, &num_elements, "array", "Memory for dynamic array.");
```

Comment:
OK! Thank you! I will try it before next push. I'm working on the DynamicObjectArray refactor, so I can test if all these implementation on DynamicArray can be implemented on DynamicObjectArray also. Also can helps me polish this pr at the same time